crates/starknet-devnet/tests/test_estimate_fee.rs

```diff
@@ -676,4 +678,202 @@ mod estimate_fee_tests {
             _ => panic!("Unexpected error: {err}"),
         };
     }
+
+    #[tokio::test]
+    async fn estimate_fee_of_declare_and_deploy_via_udc_returns_index_of_second_transaction_when_executed_with_non_existing_method()
```

Comment:
Duplicate candidate of `estimate_fee_of_multiple_txs_with_second_failing` (side note: IMO more concise names are better)

Reply:
[e4253e4](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/632/commits/e4253e45f1d20e5bad894e28361bea8ef316226d)